### ISSUE TITLE
[IDEA] Alternative FE version with Redux

### ISSUE THREAD

**Issue by Abdizriel** [AnomalyInnovations/serverless-stack-com#80]
Thanks @jayair for a great tutorial!

Did you also thought about doing alternative version of this tutorial with Redux?

**Comment by jayair** [comment]
Yeah we have! I just wanted to exclude Redux for folks that are new to React. But if there is enough interest I'll definitely cover it.

**Comment by jasjuang** [comment]
I am interested!

**Comment by jasonrundell** [comment]
Interested!

**Comment by loxtank** [comment]
Great tutorial!
I'm interested in a Redux version too.

**Comment by gregt590** [comment]
Would love to see a Redux version as well.  In the mean time I am attempting to change to a Redux-based approach myself.

**Comment by jayair** [comment]
Sounds good. If anybody has a working version with Redux, we'd love to feature it here.

**Comment by QuantumInformation** [comment]
I could be a lot of work to keep this uptodate with redux.

**Comment by jayair** [comment]
Yeah I agree, I think doing an "Extra Credit" chapter on it could work or we could link to a repo that has a Redux version put together.

**Comment by bharloe** [comment]
It doesn't exist AFAIK

**Comment by jayair** [comment]
Yeah there isn't one yet. We might add a chapter on Redux but having a separate Redux version could make things a lot more complicated.
